src/bin/facilitator-init.ts

```diff
@@ -5,7 +5,10 @@ import Account from '../Account';
 import { Chain, FacilitatorConfig } from '../Config/Config';
 import DatabaseFileHelper from '../DatabaseFileHelper';
 import Logger from '../Logger';
-import Utils from '../Utils';
+import MosaicConfig from '../Config/MosaicConfig';
+// import { Config, ENV_WORKER_PASSWORD_PREFIX } from '../Config/Config';
+// import Repositories from '../repositories/Repositories';
+// import SeedData from '../SeedData';
```

Comment:
Remove commented code.

Reply:
This code will be used to populate seedData. We can not this open up till we have dev chains and our integration test don't start using it.